src/legacy/core_plugins/data/public/query/query_bar/components/query_bar_top_row.test.tsx

```diff
@@ -121,7 +121,7 @@ describe('QueryBarTopRowTopRow', () => {
     jest.clearAllMocks();
   });
 
-  it('Should render the given query', () => {
+  it.skip('Should render the given query', () => {
```

Comment:
@lizozom I'm not sure if you're the right person to ping about this (blame `git blame` 😅), but I had to skip these tests as they were producing unhandled promise rejections due to an uncaught exception.